src/main/java/uk/gov/hmcts/darts/common/entity/EventHandlerEntity.java

```diff
@@ -38,11 +41,18 @@ public class EventHandlerEntity {
     @Column(name = "handler")
     private String handler;
 
+    @Column(name = "active")
+    private Boolean active;
+
     @CreationTimestamp
     @Column(name = "created_ts")
     private OffsetDateTime createdTimestamp;
 
     @UpdateTimestamp
     @Column(name = "last_modified_ts")
     private OffsetDateTime lastModifiedTimestamp;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "last_modified_by")
+    private UserAccountEntity lastModifiedBy;
```

Comment:
Please can this Entity be made to `implements JpaAuditing` like ExternalObjectDirectoryEntity.
In hope that we can start to standardise the entities that have these 3 fields.
It probably makes sense to adopt the field names here?

Reply:
done